lab_tests/parse_daily_tests.py

```diff
@@ -3,239 +3,258 @@
-
-xy["tests.lab.golnik.performed.todate"] = xy["tests.lab.golnik.performed"].fillna(value=0)
-xy["tests.lab.golnik.performed.todate"] = xy["tests.lab.golnik.performed.todate"].cumsum()
-xy["tests.lab.golnik.positive.todate"] = xy["tests.lab.golnik.positive"].fillna(value=0)
-xy["tests.lab.golnik.positive.todate"] = xy["tests.lab.golnik.positive.todate"].cumsum()
-
-xy["tests.lab.nlzohkp.performed.todate"] = xy["tests.lab.nlzohkp.performed"].fillna(value=0)
-xy["tests.lab.nlzohkp.performed.todate"] = xy["tests.lab.nlzohkp.performed.todate"].cumsum()
-xy["tests.lab.nlzohkp.positive.todate"] = xy["tests.lab.nlzohkp.positive"].fillna(value=0)
-xy["tests.lab.nlzohkp.positive.todate"] = xy["tests.lab.nlzohkp.positive.todate"].cumsum()
-
-xy["tests.lab.nlzohnm.performed.todate"] = xy["tests.lab.nlzohnm.performed"].fillna(value=0)
-xy["tests.lab.nlzohnm.performed.todate"] = xy["tests.lab.nlzohnm.performed.todate"].cumsum()
-xy["tests.lab.nlzohnm.positive.todate"] = xy["tests.lab.nlzohnm.positive"].fillna(value=0)
-xy["tests.lab.nlzohnm.positive.todate"] = xy["tests.lab.nlzohnm.positive.todate"].cumsum()
-
-xy["tests.lab.sbce.performed.todate"] = xy["tests.lab.sbce.performed"].fillna(value=0)
-xy["tests.lab.sbce.performed.todate"] = xy["tests.lab.sbce.performed.todate"].cumsum()
-xy["tests.lab.sbce.positive.todate"] = xy["tests.lab.sbce.positive"].fillna(value=0)
-xy["tests.lab.sbce.positive.todate"] = xy["tests.lab.sbce.positive.todate"].cumsum()
-
-xy["tests.lab.nlzohkr.performed.todate"] = xy["tests.lab.nlzohkr.performed"].fillna(value=0)
-xy["tests.lab.nlzohkr.performed.todate"] = xy["tests.lab.nlzohkr.performed.todate"].cumsum()
-xy["tests.lab.nlzohkr.positive.todate"] = xy["tests.lab.nlzohkr.positive"].fillna(value=0)
-xy["tests.lab.nlzohkr.positive.todate"] = xy["tests.lab.nlzohkr.positive.todate"].cumsum()
-
-xy["tests.lab.sbng.performed.todate"] = xy["tests.lab.sbng.performed"].fillna(value=0)
-xy["tests.lab.sbng.performed.todate"] = xy["tests.lab.sbng.performed.todate"].cumsum()
-xy["tests.lab.sbng.positive.todate"] = xy["tests.lab.sbng.positive"].fillna(value=0)
-xy["tests.lab.sbng.positive.todate"] = xy["tests.lab.sbng.positive.todate"].cumsum()
-
-xy["tests.lab.sbsg.performed.todate"] = xy["tests.lab.sbsg.performed"].fillna(value=0)
-xy["tests.lab.sbsg.performed.todate"] = xy["tests.lab.sbsg.performed.todate"].cumsum()
-xy["tests.lab.sbsg.positive.todate"] = xy["tests.lab.sbsg.positive"].fillna(value=0)
-xy["tests.lab.sbsg.positive.todate"] = xy["tests.lab.sbsg.positive.todate"].cumsum()
-
-xy["tests.lab.nlzohms.performed.todate"] = xy["tests.lab.nlzohms.performed"].fillna(value=0)
-xy["tests.lab.nlzohms.performed.todate"] = xy["tests.lab.nlzohms.performed.todate"].cumsum()
-xy["tests.lab.nlzohms.positive.todate"] = xy["tests.lab.nlzohms.positive"].fillna(value=0)
-xy["tests.lab.nlzohms.positive.todate"] = xy["tests.lab.nlzohms.positive.todate"].cumsum()
-
-xy = xy[
-    [
+
+def parse_daily_tests(xlsx=None):
```

Comment:
set default xlsx param to 
`"/data/Dnevno laboratorijsko poročilo o testiranju na okužbo s SARS-CoV-2 za <now-1>.xlsx"`
where <now-1> is yesterday's date

Reply:
I'm thinking more in terms of just giving it a folder and it finds the latest file. If you want a specific date, you can pass it an xlsx parameter.

Reply:
fine by me